cadt/vars/aarch64.yml

```diff
@@ -0,0 +1,2 @@
+---
+chia_repo_arch: "arm64"
```

Comment:
This should probably be `cadt_repo_arch`

Reply:
This is fixed in https://github.com/Chia-Network/ansible-roles/pull/256/commits/e1eae4a9a2c33e62e8cce524391017a3af07a9fd